src/main/java/com/synopsys/integration/detect/configuration/DetectProperties.java

```diff
@@ -70,7 +70,6 @@
 
 // java:S1192: Sonar wants constants defined for fromVersion when setting property info.
 // java:S1123: Warning about deprecations not having Java doc.
-@SuppressWarnings({ "java:S1123", "java:S1192" })
```

Comment:
Your formatter is incorrectly removing this warning suppression. Please consult the Save-Actions section of Engineer-Onboarding